src/I18n/I18n.php

```diff
@@ -246,6 +238,20 @@ public static function defaultLocale()
     }
 
     /**
+     * Set default fallback domain. If a domain can't provide a translation
+     * messages from the fallback domain are used.
+     *
+     * Get the currently set domain if argument is null.
+     *
+     * @param string|null $name name of the fallback-translator
+     * @return string
+     */
+    public static function defaultFallbackDomain($name = null)
```

Comment:
Threaded conversations FTW! :wink: As I said, I wasn't clear: I don't meant to drop it completely, just to remove the setter from *my* PR-code.

I updated the PR with those changes.